src/test/bundle_test.ts

```diff
@@ -291,12 +291,6 @@ suite('BuildBundler', () => {
         {
           root: path.resolve('test-fixtures/bundle-project'),
           entrypoint: 'index.html',
-          sources: [
```

Comment:
what's up here? I guess the idea is that the entrypoint is enough to get the rest of the files?